src/components/Timer/Timer.tsx

```diff
@@ -107,14 +102,21 @@ export const Timer = (props: TimerProps) => {
 
   return (
     <div id="timer" className={classNames("timer", {"timer--expired": timeLeft.m === 0 && timeLeft.s === 0})}>
+      <div className="vote-display__progress-bar" style={{right: `calc(72px - ${elapsedTimePercentage} * 72px)`}} />
       <span>
         {String(timeLeft!.m).padStart(2, "0")}:{String(timeLeft!.s).padStart(2, "0")}
       </span>
       {isModerator && (
-        <button onClick={() => store.dispatch(Actions.cancelTimer())} title={t("Timer.stopTimer")}>
-          <CloseIcon />
-        </button>
+        <div className="timer__short-actions">
+          <div className="short-actions__button-wrapper">
+            <button onClick={() => store.dispatch(Actions.cancelTimer())}>
+              <CancelIcon />
+            </button>
+            <span>{t("VoteDisplay.finishActionTooltip")}</span>
+          </div>
+        </div>
```

Comment:
are both wrappers needed here?

Reply:
yes, there will be a second short action soon